client/js/components/pages/profile.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PageWrapper from '../page-wrapper';
+import Card from 'material-ui/lib/card/card';
+import CardText from 'material-ui/lib/card/card-text';
+import CardHeader from 'material-ui/lib/card/card-header';
+import CardTitle from 'material-ui/lib/card/card-title';
+import Avatar from 'material-ui/lib/avatar';
+import Colors from 'material-ui/lib/styles/colors';
+import Paper from 'material-ui/lib/paper';
+import {generateColor} from '../mixins/color-generate';
+import ProfileStore from '../../stores/profile-store';
+import ProfileActions from '../../actions/profile-actions';
+import SnippetsList from '../snippets/snippets-list';
+
+export default class Profile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = this.getPropsFromStores();
+    this._onChange = this._onChange.bind(this);
+  }
+
+  componentDidMount(){
+    this.storeListeners = [];
+    this.storeListeners.push(ProfileStore.listen(this._onChange));
+    var reg = new RegExp('profiles\/([0-9]+)');
+    var userId = (this.props.location.pathname).match(reg)[1];
```

Comment:
@dwoznicka I am sure we should get that information from the router without parsing location